.github/workflows/integration_test_calamari.yml

```diff
@@ -0,0 +1,826 @@
+name: calamari integration test
+on:
+  push:
+    branches: [manta]
+  pull_request:
+    branches: [manta]
+    types: [opened, reopened, synchronize, labeled]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+env:
+  AWS_INSTANCE_SSH_PUBLIC_KEY: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPb24HEQ++aNFWaqVyMtIs6GotUB8R+q61XOoI2z6uMj
+  AWS_REGION: us-east-1
+  AWS_SUBNET_ID: subnet-08c26caf0a52b7c19
+  AWS_SECURITY_GROUP_ID: sg-0315bffea9042ac9b
+  AWS_INSTANCE_TYPE: c5.9xlarge
+  AWS_INSTANCE_ROOT_VOLUME_SIZE: 32
+  AWS_IMAGE_SEARCH_PATTERN: ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*
+  AWS_IMAGE_SEARCH_OWNERS: '["099720109477"]'  # canonical
+jobs:
+  get-rust-versions:
+    if: contains(github.event.pull_request.labels.*.name, 'A-calamari')
```

Comment:
actually, do not merge yet, i think the if needs an OR section that makes sure all tests run on push to manta, aka. when the PR is merged